auslib/admin/base.py

```diff
@@ -51,6 +52,7 @@ def add_security_headers(response):
 app.add_url_rule("/rules", view_func=RulesAPIView.as_view("rules"))
 # Normal operations (get/update/delete) on rules can be done by id or alias...
 app.add_url_rule("/rules/<id_or_alias>", view_func=SingleRuleView.as_view("rule"))
+app.add_url_rule("/rules/column/<column>", view_func=SingleRuleColumnView.as_view("rule_columns"))
```

Comment:
This should be "/rules/columns/<column>" - collections are plural, not singular.